src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -444,6 +444,7 @@ private async Task ShowHttpSourcesError()
                         {
                             await _logger.LogAsync(RestoreLogMessage.CreateError(NuGetLogCode.NU1302,
                             string.Format(CultureInfo.CurrentCulture, Strings.Error_HttpSource_Single, "restore", source.Source)));
```

Comment:
Since this is looping through all sources, should we have the code log an error for each one and then return true/false at the end?  Right now it fails after seeing the first one.

Reply:
🤦    thanks for noticing!